protoc-gen-gogo/generator/helper.go

```diff
@@ -99,7 +99,10 @@ func (this *pluginImports) GenerateImports(file *FileDescriptor) {
 type Single interface {
 	Use() string
 	IsUsed() bool
-	Generate() string
+	Name() string
+	Location() string
+
+	Generate() string // DEPRECATED
```

Comment:
Please delete this method in the interface and in the implementation.
I don't like deprecated code lying around its not a necessity.